### DISCUSSION TITLE
Who's Hiring (March 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#63292] [Show and tell]
# What is this?

This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

## Companies

Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

### Sample Job Post

Here's a good example of a job post.

> Vercel | Software Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers#positions)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our software engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍 

If you're a developer looking to get hired, please use #63293 instead.

Several companies have [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result!

**Comment by ThePerryR** [comment]
FTL Finance | Junior Software Engineer | St Charles, MO (Remote is possible) | Full-time | $75-95k base + up to 10% bonus | [Link](https://lewis-and-clark-partners2.prismhr-hire.com/job/747949/junior-software-engineer)

FTL Finance is lending company that focuses on financing HVAC repairs. We also have a SaaS platform that we've built that other lenders can use. The company is a little over 50 people, but the software engineering team is just me and one other person. We work with Next.js, Typescript, Tailwind, and MySQL. And our apps are mostly hosted on Vercel.

The company has been maturing it's technology over the past several years, and it's been really exciting to be a part of shaping that. And if you join our team you will also have the ability to make a large impact on the company, and learn a lot. If you love building web apps, please checkout the link!

Here's a overview of the SaaS platform we're building https://www.apptracker.dev/

**Reply by Khaan25** [reply]
Hey Perry, I'm excited to collaborate on crafting exceptional UI/UX experiences. Passionate about enhancing user satisfaction because ultimately, their happiness is our success. Ready to contribute my skills and creativity to your team.

[Portfolio](https://zzia.vercel.app/)
[LinkedIn](https://www.linkedin.com/in/ziawebdev/)
Email: zia@vennsol.pk | ziaurzai@gmail.com

P.S. Careers page doesn't work on your SaaS plateform.

**Reply by mazanat** [reply]
@ThePerryR Hey, I am a full-stack developer with 5+ yrs of exp. I am interested in this position. I'd like to contribute this project. If you are interested in my profile, please reach out to me via email. Github: fullDev9233. My email is mazanatolii9300@gmail.com. Thanks

**Reply by andentx** [reply]
Exactly the opportunity I'm looking for, just sent an application. Thanks for posting!

**Reply by nate-vukovich** [reply]
Hi Perry.

Would love to apply and I wanted to clear up if I can since I'm from Austria. But wanting/willing to relocate to the US.

Let me know.

Cheers.

**Comment by fofiu** [comment]
[Monogram](http://monogram.io/) | Senior Next.js Engineer | Remote (North & South America preferred) | Full-time | https://monogram.io/careers

**Introduction**

You don’t work at Monogram to remove roadblocks, you work here to pave new paths. You don’t just solve problems, you join a family of designers & developers who collaborate like clockwork, rise together, fall together, and do their life’s work here.

As part our strong, fast moving team that’s been a [Vercel partner](https://vercel.com/experts) from early days, you will be helping convert beautifully designed web applications into reality, including building performant APIs and wiring modern front-end using Next.js.

Your role also includes coordinating with the teams responsible for other layers of the application design & infrastructure.

**Duties**

- Collaborate on a daily basis with a team of product managers, engineers, and UI/UX designers to understand business requirements, user experience goals and pain points
- Ability to develop websites/web apps and backend APIs
- Have a strong understanding of Core Web Vitals and how code impacts performance
- Build reusable components and libraries for future use

**🚨 Required Skills**

- TypeScript
- Next.js / React / Preact
- Solid understanding of the Next.js app router and middleware concepts in Next.js
- GraphQL & REST API concepts and implementation patterns
- Testing frameworks such as Jest, Playwright, etc.
- Auth0 and other authentication services
- Content modeling in CMS’es such as Contentful, Dato, Sanity, Prismic, etc.
- Quality/clean code
- 2+ years of experience working in a fast-paced development environment using React and any of the following (Svelte, Angular, Vue, etc.)

**🧁 Bonus**
- SvelteKit 🧁🧁🧁
- A/B testing implementation
- PostgteSQL/RDBMS systems
- Understanding of API Security

**💎 Even better**

- Python and some data science concepts
- Three.js


**💭 What’s not required**

- HTML/CSS perfection — usually HTML/CSS screens will be provided by a front-end developer focused solely on perfecting the UI in HTML/CSS


If interested, please send us an email with your resume (and/or a video introduction) to [careers@monogram.io](mailto:careers@monogram.io)

**Reply by nate-vukovich** [reply]
Hi Claudio.

I'm interested and the question is if I can apply from Austria? And if there's a future to relocate to the US?

Have a good one.

**Reply by omotayosam** [reply]
hi @cfofiu do you accept interns, if you do i would love to apply for an internship role

**Comment by nellystrohmer** [comment]
Sentry | Software Engineer (React, Next.js) | Vienna, Austria | Full-time | EUR 75-90k base | Link: https://grnh.se/c0b764961us

Bad software is everywhere, and we’re tired of it. Sentry is on a mission to help developers write better software faster, so we can get back to enjoying technology.

With more than $217 million in funding and 90,000 organizations that believe we’re on to something, we're building performance and error monitoring tools that help companies and Devs spend less time fixing bugs and more time building products. If you like to selfishly build things that make your digital life better, come help us build the next generation of software monitoring tools.

I've been with the company for over two years now, continuously discovering new avenues to make a meaningful contribution, and I'm genuinely impressed by the caliber of my colleagues at Sentry. If you're interested, you can apply through the provided link. 

Consider us for the ride and learn from folks known in the industry. Armin Ronacher, the mastermind behind the Flask framework, and Yagiz, renowned as a key member of the Node.js Technical Steering Committee work with us and you could be our next Sentaur!

https://grnh.se/c0b764961us

**Reply by mino9421** [reply]
your link is not working, I would like to connect. I am based in Canada and willing to relocate

**Reply by nellystrohmer** [reply]
Job link here -> https://grnh.se/c0b764961us

**Reply by nate-vukovich** [reply]
Hi.

I'm interested and I have 2 open questions:

1. is the on-site presence a must? I live in Bleiburg, Carinthia ( 3,5h from Vienna ) -> would love to come by from time to time but mainly remote.

2. Is there a possibility to move to the US in the future? Would you sponsor a relocation to the US?

Thanks for taking the time and responding.

Cheers.

**Reply by nellystrohmer** [reply]
Hi @nvwebd !

Thanks for your questions. 

1. Yes, this is a in-person position. However, the frequency for office visits can be further aligned with the manager of the role provided the candidate is in Austria

2. No. We don't anticipate internal mobility for this role. 

All the best,
Nelly

**Reply by omotayosam** [reply]
Hi, is there an internship position i can apply to

**Reply by nellystrohmer** [reply]
Hi @omotayosam! No. Our internship positions have been filled.

**Reply by omotayosam** [reply]
@nellystrohmer thanks for the feedback. i don't mind interning for free. i just need somewhere i can learn from experienced developers so that i can gain working experience and contribute also

**Reply by nellystrohmer** [reply]
@omotayosam this is not aligned with how we operate. Candidates must apply for open positions and as of now, there are none for internships.

**Reply by DevyanshKalra** [reply]
Just submitted my application, looking forward to your response!

**Comment by hahnbeelee** [comment]
Mintlify | Full Stack Engineer | In-Person San Francisco | Full-time | $100k-$150k base | [Link](https://www.ycombinator.com/companies/mintlify/jobs/pk3AAR7-full-stack-engineer)

[Mintlify](https://mintlify.com/) is hiring a full-stack engineer. We utilize Next.js and Vercel to power thousands of company's documentation. If a company wants to create guides, tutorials, or API references, Mintlify serves as a toolkit to help them write, maintain, customize the look and feel, and host the documentation.

I'm one of the co-founders and the team consists of intrinsically motivated individuals. We like to have fun and work hard. If you love Next.js and innovating as much as I do, then please apply using the link!

**Reply by nate-vukovich** [reply]
@hahnbeelee just wanted to let you know I've applied for the position :)

**Reply by omotayosam** [reply]
hi, @hahnbeelee i am interested in an internship role
